Specie/doc/spec.md

```diff
@@ -0,0 +1,25 @@
+Entity: Specie
+=======================
```

Comment:
Definitions here should be included in the schema. See contribution manual pages 8 and 9.
[https://bt.ly/contribution_manual](https://bt.ly/contribution_manual)